wafer/talks/views.py

```diff
@@ -51,6 +52,7 @@ def get_object(self, *args, **kwargs):
         '''Only talk owners can see talks, unless they've been accepted'''
         object_ = super(TalkView, self).get_object(*args, **kwargs)
         if object_.can_view(self.request.user):
+            object_.private_notes = mark_safe(object_.private_notes)
```

Comment:
If we make it a markdown field, then it can have the same HTML permitted in the talk abstract. Presumably that's sufficient?